source/adios2/operator/compress/CompressBlosc.cpp

```diff
@@ -166,32 +166,31 @@ size_t CompressBlosc::Operate(const char *dataIn, const Dims &blockStart,
 
     if (!useMemcpy)
     {
-        const int result = blosc_set_compressor(compressor.c_str());
+        const int result = blosc1_set_compressor(compressor.c_str());
         if (result == -1)
         {
             helper::Throw<std::invalid_argument>(
                 "Operator", "CompressBlosc", "Operate",
                 "blosc library linked does not support compressor " +
                     compressor);
         }
-        blosc_set_nthreads(threads);
-        blosc_set_blocksize(blockSize);
+        blosc2_set_nthreads(threads);
+        blosc1_set_blocksize(blockSize);
```

Comment:
Is this the name of their function or ours? If ours I would not use 1, it reads a bit confusing